src/vroom_errors.h

```diff
@@ -83,15 +85,12 @@ class vroom_errors {
   void warn_for_errors() const {
     if (!have_warned_ && rows_.size() > 0) {
       have_warned_ = true;
-      static auto warn = Rf_findFun(
-          Rf_install("warn"),
-          Rf_findVarInFrame(R_NamespaceRegistry, Rf_install("rlang")));
-      cpp11::sexp warn_call = Rf_lang3(
-          warn,
-          Rf_mkString(
-              "One or more parsing issues, see `problems()` for details"),
-          Rf_mkString("vroom_parse_issue"));
-      Rf_eval(warn_call, R_EmptyEnv);
+      static auto cli_warn = cpp11::package("cli")["cli_warn"];
+      cpp11::strings bullets({
+        "w"_nm = "One or more parsing issues, call `problems()` on your data frame for details, e.g.:",
```

Comment:
Since we have the full power of cli at our disposal, we can lean in to it even more. This way we're requesting to style `problems()` "like a function" and leaving the details to cli:

```suggestion
        "w"_nm = "One or more parsing issues, call {.fun problems} on your data frame for details, e.g.:",
```